modules/install-sno-generate-the-discovery-iso.adoc

```diff
@@ -0,0 +1,51 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="generate-the-discovery-iso_{context}"]
+
+= Generate the discovery ISO
+
+Single node OpenShift installs with a discovery ISO, which the Assisted Installer (AI) generates with the cluster name, base domain, SSH public key, and pull secret.
+
+.Procedure
+
+. On the administration node, open a browser and navigate to link:https://cloud.redhat.com/openshift/assisted-installer/clusters[Install OpenShift with the Assisted Installer].
+
+. Click *Create New Cluster* to create a new cluster.
+
+. In the *Cluster Name* field, enter a name for the cluster.
+
+. In the *Base Domain* field, enter a base domain. For example:
++
+----
+example.com
+----
++
+All DNS records must be subdomains of this base domain and include the cluster name. You cannot change the base domain after cluster installation. For example:
++
+----
+<cluster-name>.example.com
+----
+
+. Select *"I want to install single node OpenShift (single node OpenShift)"*.
+
+. Read the limitations of single node OpenShift.
```

Comment:
"Read the limitations for installing on a single node."

I'll stop with the suggestions.  If you are stuck for an idea on making a replacement, please contact me.